packages/testlab/README.md

```diff
@@ -2,12 +2,26 @@
 
 A collection of test utilities we use to write LoopBack tests.
 
-## `expect`
+## Overview
 
-[Should.js](https://shouldjs.github.io/) configured in "as-function" mode
-(global `Object.prototype` is left intact) with an extra chaining word `to`.
+Test utilities to help writing loopback-next tests:
+- `expect` - behavior-driven development(BDD) style assertions
+- `sinon`
+ - stubs and mocks
+ - test spies: A function that records arguments, return value, the value of `this` and exception thrown for all its calls
```

Comment:
While we are expanding `sinon` description, let's describe stubs and mocks for consistency.

```markdown
- test spies: functions recording arguments and other information for all of their calls
- stubs: functions (spies) with pre-programmed behavior
- mocks: fake methods (like spies) with pre-programmed behavior (like stubs) as well as pre-programmed expectations
```